### PR TITLE
chore: pin foundry version in CI

### PR DESCRIPTION
**Pull request by TomAFrench**
# Description

## Problem\*

Resolves 

## Documentation\*

Check one:
- [x] No documentation needed.
- [ ] Documentation included in this PR.
- [ ] **[For Experimental Features]** Documentation to be submitted in a separate PR.

# PR Checklist\*

- [x] I have tested the changes locally.
- [x] I have formatted the changes with [Prettier](https://prettier.io/) and/or `cargo fmt` on default settings.

### CODE REVIEW

File: .github/workflows/test-js-packages.yml [hunk 1]
```diff
@@ -478,6 +478,9 @@ jobs:
 
       - name: Install Foundry
         uses: foundry-rs/foundry-toolchain@v1.2.0
+        with:
+          version: nightly-8660e5b941fe7f4d67e246cfd3dafea330fb53b1
```
Comment (aakoshh) [inline .github/workflows/test-js-packages.yml:482]: Shouldn't we use the same commit as [aztec-packages](https://github.com/AztecProtocol/aztec-packages/blob/8835b31d76b2f7c45416eaf67a748d8df9dbc753/bootstrap.sh#L79)? Although it did require a different Rust version than the repo, so if this work then great.

**Review by aakoshh — APPROVED** [review]
Does this have something to do with [this failure](https://github.com/noir-lang/noir/actions/runs/12066976367/job/33649104119)?

File: .github/workflows/test-js-packages.yml (hunk 1 above)
Comment (TomAFrench) [reply .github/workflows/test-js-packages.yml:482]: Nah, there's no need to align with them on this.

**Comment by TomAFrench** [comment]
Yep, this PR fixes that error.

**Pull request merged**